python/semantic_kernel/connectors/ai/open_ai/services/open_ai_chat_completion_base.py

```diff
@@ -155,6 +155,8 @@ def _create_chat_message_content(
         items.extend(self._get_function_call_from_chat_choice(choice))
         if choice.message.content:
             items.append(TextContent(text=choice.message.content))
+        elif hasattr(choice.message, "refusal") and choice.message.refusal:
```

Comment:
Is it always true that `message.content` and `refusal` are mutually exclusive?

Reply:
Based on the new structured output, yes. If there is a refusal returned, there is no `message.content`.